src/AzureOpenAIProxy.Management/Components/Pages/EventAdd.razor.cs

```diff
@@ -14,12 +11,34 @@ public partial class EventAdd : ComponentBase
     [Inject]
     public required NavigationManager NavigationManager { get; set; }
 
+    [Inject]
+    public required IAuthService AuthService { get; set; }
+
     [Inject]
     public required AuthenticationStateProvider AuthenticationStateProvider { get; set; }
 
+    [Inject]
+    public IModelService ModelService { get; set; } = null!;
+
+    public EventEditorModel Model { get; set; } = new();
+
+    protected override async Task OnInitializedAsync()
+    {
+        Model.AvailableModels = await ModelService.GetOwnerCatalogsAsync();
+        (Model.OrganizerEmail, Model.OrganizerName) = await AuthService.GetCurrentUserEmailNameAsync();
+        Model.Start = DateTime.Today.AddHours(-12); // allow for tz relative to utc
```

Comment:
How do we know that 12 is always relative to UTC?

Reply:
// The container time in UTC 0. 
        // Start date should be yesterday in all time zones, end date should be 7 days from now.
        Model.Start = DateTime.Today.AddDays(-1);